### PR TITLE
Branch 6 js

### PR DESCRIPTION
**Pull request by irina-tsedryk**
@burymm review please.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,10 +3,11 @@
 <head>
  
 <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
- <title>JS</title>
+<title>JS</title>
 <meta name="keywords" content="" />
 <link href="css/style.css" type="text/css" rel="stylesheet"  media="screen" />
- 
+<script type="text/javascript" src="js/js.js"></script>
```
Comment (burymm) [inline index.html]: 1) You includes this fils twice. One of them should be remove.
2) You should put js scripts on the bottom of page. I should load after dom loaded. It's best practice.

File: index.html [hunk 2]
```diff
@@ -42,21 +42,71 @@
...
 
 		<aside class="left-sidebar">
 			<p>COMPANY</p>
 			<ul>
-				<li><a href="#">About Us</a></li>
-				<li><a href="#">Management</a></li>
-				<li><a href="#">Investor Relations</a></li>
-				<li><a href="#">Distributors</a></li>
-				<li><a href="#">Contact Us</a></li>
+				<li><a href="#" onclick="myFunction_About()">About Us</a></li>
```
Comment (burymm) [inline index.html]: Write js in html bad practice I will create task to fix it.

File: js.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+function myFunction_About()
```
Comment (burymm) [inline js.js]: I created taks to write different code. It hard task we need separate it.

File: index.html (hunk 1 above)
Comment (irina-tsedryk) [reply index.html]: @burymm I shifted the script and remove unnecessary. while the rest don't know what to do.

File: index.html (hunk 2 above)
Comment (irina-tsedryk) [reply index.html]: @burymm, I look forward to new tasks)

File: js.js (hunk 3 above)
Comment (irina-tsedryk) [reply js.js]: @burymm I don't understand what do you mean

Comment (burymm) [reply js.js]: Please look at my branch. I updated your code. Then  you can merge my code into your branch. Just click merge pull request.

File: index.html [hunk 4]
```diff
@@ -123,7 +121,7 @@
 			<img src="images/footer_logo.png" alt="" title="" />
 		</div>
 	</footer><!-- .footer -->
-
 </div><!-- .wrapper -->
+<script src="js/js.js"></script>
```
Comment (burymm) [inline index.html]: You don't need now js.js. We will use main.js. You can remove that.

**Comment by irina-tsedryk** [comment]
привет, я вроде удалила его. у меня в ветке 6 нет файла js.js

2017-01-28 22:40 GMT+03:00 Mikalai <notifications@github.com>:

**Comment by burymm** [comment]
You have conflicts. You need merge with master.

**Comment by burymm** [comment]
and we should update this branch from master.

**Pull request merged**